src/main/java/com/nike/cerberus/cli/EnvironmentConfigToArgsMapper.java

```diff
@@ -86,13 +80,36 @@ private EnvironmentConfigToArgsMapper() {
         }
 
         // convert to string array and return
-        return args.toArray(new String[args.size()]);
+        return args.toArray(new String[0]);
```

Comment:
This is probably no necessary but doesn't hurt either.

Reply:
Intellij did that, I can revert it.

Reply:
I don't mind either way

Reply:
- [ ] revert this.